### ISSUE TITLE
Weird line wrap issue on Track overview.

### ISSUE THREAD

**Issue by joshgoebel** [exercism/v3-beta#13]
<img width="482" alt="Screen Shot 2021-05-17 at 8 32 09 AM" src="https://user-images.githubusercontent.com/6473/118504291-60147f00-b6f9-11eb-9dcd-52a11dfe2108.png">

Perhaps needs non-breaking dash? `&#8209;` per KOTP.

**Comment by iHiD** [comment]
@joshgoebel Is this because your screen was too small to fit it on?

**Comment by joshgoebel** [comment]
Yes it was a width issue, but it wasn't SUPER small by any stretch.

**Comment by iHiD** [comment]
Yeah, so we have **no** responsive support atm. If you're less than 1100px, we're not interested in your joy 😉
